applications/StructuralMechanicsApplication/tests/test_patch_test_large_strain.py

```diff
@@ -49,7 +49,7 @@ def _apply_BCs(self,mp,A,b):
             
             node.SetSolutionStepValue(KratosMultiphysics.DISPLACEMENT,0,u)
             
-    def _define_movement(self,dim):
+    def _define_movement_TL(self,dim):
```

Comment:
I wanted to have the PR to have some feedback while I am still working in the topic